daemon/mgr/container.go

```diff
@@ -989,6 +989,13 @@ func (mgr *ContainerManager) Update(ctx context.Context, name string, config *ty
 	}
 	c.Unlock()
 
+	// Update Env
```

Comment:
update container parameters, should we add `Lock` here?

Reply:
This should be disscuss in another pr, we are not determine to modify origin logic here, or you can raise a pr if  you think if lock should be added.